src/test/java/de/tum/in/www1/artemis/authentication/UserSaml2IntegrationTest.java

```diff
@@ -52,6 +52,13 @@ void clearAuthentication() {
         this.database.resetDatabase();
     }
 
+    @Test
+    void testAuthenticationRedirect() throws Exception {
+        request.postWithResponseBody("/api/saml2", Boolean.FALSE, UserJWTController.JWTToken.class, HttpStatus.UNAUTHORIZED);
+        final String redirectTarget = request.getRedirectTarget("/saml2/authenticate", HttpStatus.FOUND);
+        assertThat(redirectTarget).isEqualTo("http://localhost/login");
```

Comment:
Are tests always executed on `localhost`? Then this line is not an issue, otherwise it might be.

Reply:
Replaced it with a `endsWith` check, that definitely shouldn’t break.